### ISSUE TITLE
Fature request: Display title instead of filename

### ISSUE THREAD

**Issue by sikri-eic** [blakedietz/vscode-nested-tags#64]
This is in the same category as #49 and #51 (what to display in the "exploerer"). I really do not care about filenames and in my setup they are mostly auto-generated (datetime). But frontmatter almost always have a `title` and I think I'm not the only one with this kind of setup. So, it would be great if the "explorer" would display the title instead of filenames (and perhaps fall back to filename).

**Comment by HebelHuber** [comment]
I agree, that would be a really nice feature. I found the line responsible for the name [here](https://github.com/blakedietz/vscode-nested-tags/blob/master/src/tag-tree-data-provider.ts#L71).
